Tasks/Maven/moduleanalysis.ts

```diff
@@ -1,7 +1,13 @@
-import ar = require('./analysisresult');
+import {AnalysisResult} from './analysisresult';
+
 // Data class for supporting Maven projects with more than one module
 export class ModuleAnalysis {
     moduleName: string;
     rootDirectory: string;
-    analysisResults:any = {}; // A dictionary of toolName:string -> analysisResult:AnalysisResult
+    analysisResults:ToolAnalysisResultsDict = {};
+}
+
+// A dictionary of toolName:string -> analysisResult:AnalysisResult
+interface ToolAnalysisResultsDict {
+    [toolName:string]:AnalysisResult
```

Comment:
Additional robustness -define the types of the dictionary to be string -> AnalysisResult.